dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -440,6 +442,7 @@ export const renderElement = ({
 					switches={switches}
 					editionId={editionId}
 					RenderArticleElement={RenderArticleElement}
+					isLastElement={index >= totalElements - 1}
```

Comment:
Technically if we’re confident enough about our code…?

```suggestion
					isLastElement={index === totalElements - 1}
```